### PR TITLE
Updated Travis flow

### PR DESCRIPTION
**Pull request by kapaljaka**

### CODE REVIEW

File: fastlane/Fastfile [hunk 1]
```diff
@@ -29,12 +29,33 @@ platform :ios do
   	})
   end
 
-  desc "Notifies appropriate Slack channel when tests fails"
-  lane :report_tests_failed do 
+  lane :report_pr_coulbe_merged do
```
Comment (artemstepanenko) [inline fastlane/Fastfile]: report_pr_could_be_merged ?

File: fastlane/test_output/report.html [hunk 2]
```diff
@@ -187,7 +187,7 @@ <h3 class="time">0.001s</h3>
               <tr class="test passing odd"  onclick="toggleDetails('a_clear_method__removes_all_images');">
                 <td>
                   
-                    <h3 class="time">0.001s</h3>
+                    <h3 class="time">0.006s</h3>
```
Comment (artemstepanenko) [inline fastlane/test_output/report.html]: Should we add this file to the repo?

File: fastlane/travis_scripts/travis_after_success.sh [hunk 3]
```diff
@@ -1,3 +1,7 @@
 if [ "{$TRAVIS_BRANCH}" == "develop" ]; then
 	fastlane ios report_test_coverage
+fi
+
+if ["{$TRAVIS_PULL_REQUEST}" != 'false']; then
```
Comment (artemstepanenko) [inline fastlane/travis_scripts/travis_after_success.sh]: Isn't it more strait forward to compare TRAVIS_PULL_REQUEST to 'true'?

File: fastlane/Fastfile (hunk 1 above)
Comment (kapaljaka) [inline fastlane/Fastfile]: @artemstepanenko the fuck are you doing in 'WIP' pull-request?

File: fastlane/travis_scripts/travis_after_success.sh (hunk 3 above)
Comment (kapaljaka) [inline fastlane/travis_scripts/travis_after_success.sh]: no, because it returns commit SHA if it's PR

File: fastlane/Fastfile (hunk 1 above)
Comment (artemstepanenko) [inline fastlane/Fastfile]: :)

File: fastlane/test_output/report.html (hunk 2 above)
Comment (kapaljaka) [inline fastlane/test_output/report.html]: I think yes, we might add performance reports in the feature to be posted on slack

**Comment by kapaljaka** [comment]
@artemstepanenko could be reviewed

**Comment by artemstepanenko** [comment]
Seems ok.

**Pull request merged**